packages/constants/src/constants.ts

```diff
@@ -18,6 +18,7 @@ export enum EnumBuildingTypes {
 }
 
 export const STR_BUILDING_TYPE_ANALYSIS_ID = "building-type-analysis";
+export const STR_WALKTHROUGH_ID = "walkthrough";
```

Comment:
I honestly asked myself that same question while writing it 😂. I left it the same as the line above because they have similar uses in that they are both used for the url path. The building type string was named ID originally because it was the ID of the building type walkthrough. The other walkthroughs have their own IDs (9.9.9, 9.10.14). I think it'd be fine removing the '_ID' on both if we felt it would be less confusing.